noxfile.py

```diff
@@ -0,0 +1,247 @@
+    ],
+)
+@nox.parametrize("device", [DeviceNoxParam.cpu, DeviceNoxParam.cuda12])
+@nox.parametrize("codegen", [CodeGenNoxParam.internal, CodeGenNoxParam.dace])
+def test_next(
+    session: nox.Session,
+    codegen: CodeGenOption,
+    device: DeviceOption,
+    meshlib: Literal["nomesh", "atlas"],
+) -> None:
+    """Run selected 'gt4py.next' tests."""
+
+    codegen_settings = CodeGenTestSettings[codegen]
+    device_settings = DeviceTestSettings[device]
+    mesh_markers = []
+    match meshlib:
+        case "nomesh":
+            mesh_markers.append("not requires_atlas")
+        case "atlas":
+            mesh_markers.append("requires_atlas")
+
+    _install_session_venv(
+        session,
+        extras=["performance", "testing", *codegen_settings["extras"], *device_settings["extras"]],
+        groups=["test"],
+    )
+
+    num_processes = session.env.get("NUM_PROCESSES", "auto")
+    markers = " and ".join(codegen_settings["markers"] + device_settings["markers"] + mesh_markers)
+
+    session.run(
+        *f"pytest --cache-clear -sv -n {num_processes}".split(),
+        *("-m", f"{markers}"),
+        str(pathlib.Path("tests") / "next_tests"),
+        *session.posargs,
+        success_codes=[0, NO_TESTS_COLLECTED_EXIT_CODE],
+    )
+    session.run(
+        *"pytest --doctest-modules --doctest-ignore-import-errors -sv".split(),
+        str(pathlib.Path("src") / "gt4py" / "next"),
+        success_codes=[0, NO_TESTS_COLLECTED_EXIT_CODE],
+    )
+
+
+@nox.session(python=["3.10", "3.11"], tags=["cartesian", "next"])
+@nox.parametrize("device", [DeviceNoxParam.cpu, DeviceNoxParam.cuda12])
+def test_storage(
+    session: nox.Session,
+    device: DeviceOption,
+) -> None:
+    """Run selected 'gt4py.storage' tests."""
+
+    device_settings = DeviceTestSettings[device]
+
+    _install_session_venv(
+        session, extras=["performance", "testing", *device_settings["extras"]], groups=["test"]
+    )
+
+    num_processes = session.env.get("NUM_PROCESSES", "auto")
+    markers = " and ".join(device_settings["markers"])
+
+    session.run(
+        *f"pytest --cache-clear -sv -n {num_processes}".split(),
+        *("-m", f"{markers}"),
+        str(pathlib.Path("tests") / "storage_tests"),
+        *session.posargs,
+    )
+    session.run(
+        *"pytest --doctest-modules -sv".split(),
+        str(pathlib.Path("src") / "gt4py" / "storage"),
+        success_codes=[0, NO_TESTS_COLLECTED_EXIT_CODE],
+    )
+
+
+# -- utils --
+def _install_session_venv(
+    session: nox.Session,
+    *args: str | Sequence[str],
+    extras: Sequence[str] = (),
+    groups: Sequence[str] = (),
+) -> None:
+    """Install session packages using uv."""
+    session.run_install(
+        "uv",
+        "sync",
+        "--no-dev",
+        *(f"--extra={e}" for e in extras),
+        *(f"--group={g}" for g in groups),
+        env={key: value for key, value in os.environ.items()}
+        | {"UV_PROJECT_ENVIRONMENT": session.virtualenv.location},
```

Comment:
```suggestion
        | {"UV_PROJECT_ENVIRONMENT": session.virtualenv.location, "UV_PYTHON": session.python},
```

Reply:
Done in a different way with a command line argument.